skeleton_for_bot.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+TOKEN = 'YOUR_TOKEN' # insert your token from BotFather here
+BASE_URL = f"https://api.telegram.org/bot{TOKEN}/"
+LONG_POLLING_TIMEOUT = 10
+
+def bot_function():
+    ...
```

Comment:
If you want function that does nothing, specify `pass`  with appropriate comment in it's body - what this function should do in general.